data/sql/updates/pending_db_world/remove-garbage-sql-ai-handling-and-add-simple-sai.sql

```diff
@@ -0,0 +1,6 @@
+-- remove old SAI trying to script mob behaviour (see AC issue for old SAI) and add simple behaviour equivalent to Riverpaw Mystic
+DELETE FROM `smart_scripts` WHERE `entryorguid` = 16337 AND `source_type` = 0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(16337, 0, 0, 0, 0, 0, 100, 0, 0, 0, 3400, 4800, 0, 11, 20802, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Blackpaw Shaman - In Combat CMC - Cast \'Lightning Bolt\''),
+(16337, 0, 1, 0, 2, 0, 100, 1, 0, 30, 15000, 25000, 0, 11, 28902, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Blackpaw Shaman - Between 0-30% Health - Cast \'Bloodlust\''),
```

Comment:
1. Don't interrupt casts. That's deprecated from when casts could cancel each other out if not interrupted.
2. Are they supposed to have caster-like movement? Like standing still and not chasing the player all the time? If so use cast_flag 64